Assets/Project/Modules/VFX/Generic/Scripts/ParticleFactory/ParticleFactoryConfig.cs

```diff
@@ -8,7 +13,33 @@ namespace Popeye.Modules.VFX.ParticleFactories
     
     public class ParticleFactoryConfig : ScriptableObject
     {
-        public GameObject simpleParticlePrefab;
-        public GameObject circleParticlePrefab;
+        [System.Serializable]
+        class ParticleTypeToRecyclable
+        {
+            [SerializeField] private ParticleTypes _key;
+            [SerializeField] private RecyclableObject _value;
```

Comment:
Revisiting this, I think it would be better if the variables were named _particleType & _particlePrefab respectively, same with the properties